server/swagger/docs/html/index.html

```diff
@@ -2,7 +2,8 @@
 <html>
 <head>
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <title>Focalboard Server</title>
+  <title>line /Users/harshilsharma/codebase/mattermost/focalboard/server/main/main.go:1:1
+Package classification Focalboard Server</title>
```

Comment:
I don't know swagger, but these changes look suspicious to me. Although, I'm guessing these were done automatically.

Reply:
Yeah, you need to manually go in and clean up any hardcoded paths when re-generating the Swagger docs.

Reply:
In this case, the title should just be **Focalboard Server**, as before.

Reply:
Oh, didn't notice that swagger picked my local paths. I've fixed these now.